### ISSUE TITLE
Word Anvil Won't Save

### ISSUE THREAD

**Issue by TheMannaKing** [worldanvil/worldanvil-bug-tracker#471]
**World Anvil Username**: TheMannaKing

**World / Campaign / RPG Character / Manuscript Title**: The Manna Realms (World)

**Feature**: Saving Data on Articles

**Related URLS** (To public & visible pages add screenshots otherwise): https://www.worldanvil.com/w/the-manna-realms-themannaking/a/germaine-s-journal-item?preview=true (example)

**Describe the Issue**
A clear and concise description of what the issue is.
I am unable to save any new information I write up on your site. I can make a new article and title it, but that's as far as it goes on my information being saved. I normally use the FireFox browser with no adblockers but have tried saving on my account from both Google Chrome and Microsoft Edge to find that they don't work either.

**To Reproduce**
1. Make a new article
2. Put data within the article
3. Save Changes. See that it says "Saved"
4. Go to "view"
5. See there is no information

**Expected behavior**
The article I wrote to save.

**Screenshots**
If applicable, add screenshots to help explain your problem. You can add screenshots by drag & drop into this window!

**Comment by Kordishal** [comment]
This does not generally happen so not sure what it could be, but it is not a general bug as it usually works. Can you come to the discord to figure out what is causing this?

[World Anvil Discord](https://www.worldanvil.com/w/WorldAnvilCodex/a/discord).

My first guess would have been that it is caused by an extension, but that seems unlikely as you have tried in several browsers. But just to be save can you attempt to use a private window in Firefox and login and try and save something?

**Comment by TheMannaKing** [comment]
I'll join the discord just in case the problem happens again. It worked in a private window. Then out of curiosity before I bothered you lovely people more I tried it in my normal browser and found it's saving just fine today. I plan on simply getting work done while it lasts and popping over in discord to poke someone if it happens again. Thanks a lot for trying to help even though it seems to have resolved itself, lol.
